distributed/deploy/cluster.py

```diff
@@ -91,6 +91,15 @@ async def _close(self):
         self.status = Status.closed
 
     def close(self, timeout=None):
+        # If the cluster is already closed, we're already done
+        if self.status == Status.closed:
+            if self.asynchronous:
```

Comment:
Why is this section required? It seems like it could use testing.

Reply:
@martindurant this is covered in the discussion above.  `Cluster.close` is used in both a sync and an async manner.  If it is asynchronous then it should return an awaitable future.  If it is synchronous then it can return None.